integrations/GA4/ECommerceEventConfig.js

```diff
@@ -1,3 +1,48 @@
+const ITEM_PROP_EXCLUSION_LIST = [
```

Comment:
Can we add the link to the documentation here?

Reply:
basically, we have not derived it based on any documentation. if you take a look at `itemParametersConfigArray` in the file `integrations/GA4/ECommerceEventConfig.js`, you will see, we are already mapping some product parameters. We have created a list from there, just to avoid duplicate fields. Basically we are considering any other fields, except those, as a custom field.